core-feature-pack/common/src/main/resources/content/bin/systemd/launch.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+if [ "$WILDFLY_SYSTEMD_DEBUG" == "true" ]; then
```

Comment:
Another missing "${" that I failed to spot the first time :(

Reply:
Done